lambda/src/main/scala/migration/Main.scala

```diff
@@ -1,13 +1,39 @@
 package migration
 
 import org.flywaydb.core.Flyway
+import pureconfig._
+import pureconfig.generic.auto._
+import software.amazon.awssdk.auth.credentials.DefaultCredentialsProvider
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.rds.RdsUtilities
+import software.amazon.awssdk.services.rds.model.GenerateAuthenticationTokenRequest
+
+object B extends App {
```

Comment:
Does this let you run the migrations locally, or is it for some other purpose? Could you give it a clearer name and a comment if that seems helpful?

Reply:
Totally didn't mean to commit that. I've removed it. If you want to run the migrations locally then you can use the sbt task so there's no need to be able to run it in the code.